lua/config/autocmds.lua

```diff
@@ -0,0 +1,103 @@
+local autocmds = {}
+
+function autocmds.setup(config)
+  config:extend {
```

Comment:
Why is `git` not treating this as a rename? If it's because of this line, maybe try 

```lua
local autocmd_groups = {
  _general_settings = {
  --
  }
}
  
function autocmds.setup(config)
  config:extend(autocmd_groups)
end
```

Reply:
I don't mind using the same pattern as for plugins